ambry-api/src/main/java/com.github.ambry/network/ServerNetworkResponseMetrics.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright 2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.network;
+
+import com.codahale.metrics.Histogram;
+
+
+/**
+ * Tracks a set of metrics for a network response by a Server
+ */
+public class ServerNetworkResponseMetrics extends NetworkSendMetrics {
```

Comment:
Again, this is not just capturing metrics for a send. This captures metrics related to a response sent by the server. and hence the name